ycsb/main.go

```diff
@@ -342,6 +344,77 @@ type cockroach struct {
 	writeStmt *sql.Stmt
 }
 
+func initRelationalStrategy(db *sql.DB) (*sql.Stmt, *sql.Stmt, error) {
```

Comment:
comment needed, to explain what the three returned values are.

Reply:
actually just rip out this complexity.

im not the biggest fan of this statefulness, where it executes the create statement and then returns the read and write statements. why not just have three statically defined arrays of strings?
`createStmt[schema]`, `readStmt[schema]` ...
then a simple `schema = JSON` means does all the work for you. then you dont even need these `strategy` helper methods.

Anyway I'm not going to insist on this, but food for future thought.